frontend/src/App.tsx

```diff
@@ -45,6 +62,27 @@ function App() {
     setNewTodoDescription("");
   };
 
+  async function delTodo(url: string, id: string) {
+    return fetch(url + id, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+  }
+
+  const deleteTodo = async (id: string) => {
```

Comment:
deleteTodo and delTodo sound very similar and hence confusing. you can name the function that is called first on user click `onDeleteTodo` and the other one `deleteTodo`